src/lib/ecore/ecore_main.c

```diff
@@ -13,12 +13,17 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
-#include <unistd.h>
 #include <math.h>
-#include <sys/types.h>
 #include <errno.h>
+#include <sys/types.h>
```

Comment:
I think we should let `sys/types` where it was, I don't know if phab will like this latter.